src/tilt.directive.ts

```diff
@@ -0,0 +1,43 @@
+import * as VanillaTilt from "vanilla-tilt";
+import { OnInit, OnDestroy, Directive, Input, ElementRef, Output, EventEmitter } from "@angular/core";
+import { TiltOptions, TiltValues } from "vanilla-tilt";
+
+@Directive({
+	selector: "[tilt]",
```

Comment:
for consistency prefix this with `ssv` (i know command is not but i want to rename it)